### PR TITLE
🕊️ Remove trigger firing functionality

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/triggers/models.py [hunk 1]
```diff
@@ -84,18 +83,6 @@ class Trigger(SmartModel):
         related_name="triggers",
     )
 
-    last_triggered = models.DateTimeField(
```
Comment (rowanseymour) [inline temba/triggers/models.py]: seems we haven't been using this or `triggered_count` for a while now

Comment (nicpottier) [reply temba/triggers/models.py]: It would be a cool thing to add back in at some point but ya would need to be a squashable model.

**Comment by rowanseymour** [comment]
@nicpottier just no keeping you out of WIP PRs... 😁

**Comment by nicpottier** [comment]
New rule! How about no emoji until it's ready?

**Comment by rowanseymour** [comment]
Would that stop you? To be honest I don't expect people to review stuff unless I request a review. I like making PRs early cuz a PR a nice a diff of everything, and you have a it start Travis on the tests, make comments on the code whilst things in on your mind.

🚧

File: temba/triggers/models.py [hunk 2]
```diff
@@ -295,101 +269,24 @@ def import_triggers(cls, org, user, trigger_defs, same_site=False):
...
-
-        # if message text is only one word, then we can match 'only-word' triggers too
-        match_types = (cls.MATCH_FIRST_WORD, cls.MATCH_ONLY_WORD) if len(words) == 1 else (cls.MATCH_FIRST_WORD,)
-        trigger = trigger.filter(keyword__iexact=words[0], match_type__in=match_types)
-
-        # trigger needs to match the contact's groups or be non-group specific
-        trigger = trigger.filter(Q(groups__in=msg.contact.user_groups.all()) | Q(groups=None))
-
-        trigger = trigger.prefetch_related("groups", "groups__contacts").order_by("groups__name").first()
+    def fire_from_schedule(self):
```
Comment (rowanseymour) [inline temba/triggers/models.py]: @nicpottier for completeness seems like this stuff should make the move to mailroom right?

**Pull request merged**